dracut/30ignition/selinux-policy-persist

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# Allows for auto-updating the SELinux base policy
```

Comment:
I think the better place for these setup steps would be in `dracut/99setup-root/initrd-setup-root`

Reply:
could you elaborate?

Reply:
Sure, I can move it there. Will this still run before ignition? That's also one requirement.